samples/src/main/java/io/zeebe/example/workflow/WorkflowInstanceWithResultCreator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.example.workflow;
+
+import io.zeebe.client.ZeebeClient;
+import io.zeebe.client.ZeebeClientBuilder;
+import io.zeebe.client.api.response.WorkflowInstanceResult;
+import java.time.Duration;
+import java.util.Map;
+
+public class WorkflowInstanceWithResultCreator {
+  public static void main(final String[] args) {
+    final String broker = "127.0.0.1:26500";
+
+    final String bpmnProcessId = "demoProcessSingleTask";
+
+    final ZeebeClientBuilder builder =
+        ZeebeClient.newClientBuilder().brokerContactPoint(broker).usePlaintext();
+
+    try (final ZeebeClient client = builder.build()) {
+
+      openJobWorker(client); // open job workers so that task are executed and workflow is completed
+      System.out.println("Creating workflow instance");
+
+      final WorkflowInstanceResult workflowInstanceResult =
+          client
+              .newCreateInstanceCommand()
+              .bpmnProcessId(bpmnProcessId)
+              .latestVersion()
+              .withResult()
```

Comment:
Should we mark this line somehow ? with a comment that this is the interesting part

Reply:
Yes.